File: BNOI - Group E/Average/Average.cs
using System;

class Average
{
    static void Main()
    {
        string[] input = Console.ReadLine().Split(' ');
        int m = int.Parse(input[0]);
        int n = int.Parse(input[1]);
        Console.WriteLine((((m % 2 != 0) ? m + 1 : m) + ((n % 2 != 0) ? n - 1 : n)) / 2);
    }
}

File: BNOI - Group E/Average/Average.js
function solve(args) {
    var input = args[0].split(' ');
    var m = parseInt(input[0]);
    var n = parseInt(input[1]);
    return (((m % 2 != 0) ? m + 1 : m) + ((n % 2 != 0) ? n - 1 : n)) / 2;
}